src/components/AboutBook.js

```diff
@@ -17,7 +25,7 @@ const BookPage = ({ book }) => {
               </div>
 
                 <div className='about-btn-container'>
-                    <button className='return-book-btn'>
+                    <button className='return-book-btn'onClick={toggleScanner}>
                     <span className='btn-text'>Return book</span>

```

Comment:
Lets make it like that: when scanner is off - button name is Return book, when its on - button name is cancel and onclick function is start / stop respectively